XML/syntax_test_xml.xml

```diff
@@ -17,9 +17,17 @@
 <!--                     ^^ punctuation.definition.tag.end -->
 <!--                       ^ - meta.tag.preprocessor -->
 
-
-     <?XML version="1.0" ?>
-<!-- ^ invalid.illegal.missing-entity - meta.tag.preprocessor -->
+     <?XML VERSION="1.0" ?>
+<!-- ^^^^^^^^^^^^^^^^^^^^^^ meta.tag.preprocessor -->
```

Comment:
Wow, quick anwer. But the other tags like `<XML-STYLESHEET>` are case insensitive?